src/2018/transitioning/modules/macros.md

```diff
@@ -41,6 +41,34 @@ fn main() {
 
 This moves `macro_rules` macros to be a bit closer to other kinds of items.
 
+
+## Procedural macros
+
+When using procedural macros to derive traits, you could have written:
+
+```rust,ignore
+// Rust 2015
+extern crate serde;
+#[macro_use] extern crate serde_derive;
+
+#[derive(Serialize, Deserialize)]
+struct Bar;
+```
+
+Now, you write instead:
+
+```rust,ignore
+// Rust 2018
+#![feature(rust_2018_preview, use_extern_macros)]
```

Comment:
I think `use_extern_macros` is implied by `rust_2018_preview`, so that feature should be removable here

Reply:
I copied it from the existing docs, so I guess it is redundant there as well. I will remove both.